<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_part3_1</class>
 <widget class="QWidget" name="fct_part3_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="numIn_Suppression">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Suppression">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Suppression</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ajout">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajout</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="numIn_Ajout">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="numEp_Modification">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="numEp_Suppression">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="numIn_Modification">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Modification">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modification</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestion des inscriptions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_Suppression">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_Modification">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_Ajout">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numEp_Ajout">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titre_NumEp">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>numEp</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titre_NumIn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>numIn</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bouton_Ajout</sender>
   <signal>clicked()</signal>
   <receiver>fct_part3_1</receiver>
   <slot>ajouterRes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouton_Suppression</sender>
   <signal>clicked()</signal>
   <receiver>fct_part3_1</receiver>
   <slot>supprimerRes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numEp_Suppression</sender>
   <signal>activated()</signal>
   <receiver>fct_part3_1</receiver>
   <slot>changedSuppression()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numEp_Modification</sender>
   <signal>activated()</signal>
   <receiver>fct_part3_1</receiver>
   <slot>changedModification()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouton_Modification</sender>
   <signal>clicked()</signal>
   <receiver>fct_part3_1</receiver>
   <slot>modifierRes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>

 </connections>
 <slots>
  <slot>ajouterRes()</slot>
  <slot>supprimerRes()</slot>
  <slot>ModifierRes()</slot>
 </slots>
</ui>
